src/amo/components/HeroRecommendation/styles.scss

```diff
@@ -24,6 +24,22 @@
   }
 }
 
+.HeroRecommendation--no-image {
+  justify-content: center;
+
+  @include respond-to(extraExtraLarge) {
+    @include padding-start(36px);
+  }
+
+  .HeroRecommendation-info {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+
+    @include margin-start(0);
```

Comment:
How about this?

<details>
<summary>Patch</summary>

```diff
diff --git a/src/amo/components/HeroRecommendation/styles.scss b/src/amo/components/HeroRecommendation/styles.scss
index ee4997f74..c8ff98965 100644
--- a/src/amo/components/HeroRecommendation/styles.scss
+++ b/src/amo/components/HeroRecommendation/styles.scss
@@ -24,38 +24,6 @@
   }
 }
 
-.HeroRecommendation--no-image {
-  justify-content: center;
-
-  @include respond-to(extraExtraLarge) {
-    @include padding-start(36px);
-  }
-
-  .HeroRecommendation-info {
-    align-items: center;
-    display: flex;
-    flex-direction: column;
-
-    @include margin-start(0);
-  }
-}
-
-.HeroRecommendation-image {
-  height: 0;
-  visibility: hidden;
-  width: 0;
-
-  @include respond-to(medium) {
-    height: auto;
-    visibility: visible;
-    width: 100%;
-  }
-
-  @include respond-to(extraExtraLarge) {
-    width: 480px;
-  }
-}
-
 .HeroRecommendation-recommended {
   font-size: $font-size-s;
   letter-spacing: 0.1em;
@@ -100,6 +68,48 @@
   }
 }
 
+.HeroRecommendation--no-image {
+  justify-content: center;
+
+  @include respond-to(extraExtraLarge) {
+    @include padding-start(36px);
+  }
+
+  .HeroRecommendation-info {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    margin: 0 auto;
+
+    @include respond-to(large) {
+      width: 576px;
+    }
+  }
+
+  .HeroRecommendation-recommended,
+  .HeroRecommendation-heading,
+  .HeroRecommendation-body,
+  .HeroRecommendation-link {
+    text-align: center;
+  }
+}
+
+.HeroRecommendation-image {
+  height: 0;
+  visibility: hidden;
+  width: 0;
+
+  @include respond-to(medium) {
+    height: auto;
+    visibility: visible;
+    width: 100%;
+  }
+
+  @include respond-to(extraExtraLarge) {
+    width: 480px;
+  }
+}
+
 .HeroRecommendation-link {
   background-color: transparent;
   border: 4px solid $white;
```

</details>

* I added a width to match the mock in the issue
* I fixed centering in all text elements to match the mock

Reply:
Sure, that looks good. I also added your suggested change to switch from `136px` to `132px`.

Reply:
I realized that I was the one who originally added `136px`, hah. I probably got it from measuring the mocks. I think adding subtle consistencies does make a difference.